packages/stimulus/src/index.ts

```diff
@@ -0,0 +1,19 @@
+export function installErrorHandler(appsignal: any, application: any) {
+  const DEFAULT_ACTION = "Application.handleError"
```

Comment:
Same point I brought up in the angular integration. This makes it look like the error occurred in the error handler rather than be caught by it. https://github.com/appsignal/appsignal-javascript/pull/21#discussion_r304760884

Reply:
See my comment on #21.